features/local_diff_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require_relative '../spec/spec_helper'
+
+describe 'local diff' do
+  let(:fixtures_path) { Pathname(__dir__).join('fixtures') }
+  let(:root) { fixtures_path.join('local_diff') }
+  let(:lib_path) { root.join('lib') }
+  let(:class1_path) { lib_path.join('class1.rb') }
+  let(:class2_path) { lib_path.join('class2.rb') }
+  let(:spec_path) { root.join('spec') }
+  let(:spec_file_path) { spec_path.join('file_spec.rb') }
+  let(:spec_helper_path) { spec_path.join('spec_helper.rb') }
+
+  after do
+    fixtures_path.rmtree
+  end
+
+  def foresee
+    Crystalball.foresee(root, root.join('execution_map.yml'))
+  end
+
+  before do
+    lib_path.mkpath
+    spec_path.mkpath
+
+    class1_path.open('w') { |f| f.write <<~RUBY }
+      class Class1
+        def initialize
+          @var = 1
+        end
+      end
+    RUBY
+
+    class2_path.open('w') { |f| f.write <<~RUBY }
+      class Class2
+        def initialize
+          @var = 2
+        end
+      end
+    RUBY
```

Comment:
Maybe we can move these 2 initial files + spec file to fixtures?

Reply:
We could have them in fixtures and then copy them to some temporary folder where we would initialize git repo and do all the other things. Not sure which solution would be better current or that one.